### DISCUSSION TITLE
Electronics source code?

### DISCUSSION THREAD

**Discussion by ghost** [bassamanator/everything-sovol-sv06#3] [Q&A]
I can't find any PCB layouts or schematics or wiring loom specs or connector designations or display controller or basically anything electronic for any of it. Mechanics and firmware seem to be on their github, but that's all. 

I sent them an email requesting this stuff and will share it here if they allow it, but I'm hoping I've just missed it and it's already available? Any suggestions?

Any way, thanks! It's nice to have somewhere to discuss the printer.

**Comment by bassamanator** [comment]
Great idea! Unfortunately, I only have a picture of the hotend PCB and the official picture of the motherboard. There is *some* perhaps relevant information [here](https://www.printables.com/model/382629-sovol-sv06-skr-3-ez-mainboard-case-upgrade).

Will gladly include schematics as soon as they become available!

**Reply by ClearAirTurbulence3D** [reply]
I think the original SV06 board is a slight variation of the Creality Ender 3 32 bit motherboard.

I have an SKR3 EZ board and its extra X axis will make it a good upgrade. Thanks for posting this.

**Comment by ghost** [comment]
I have not received the schematics yet, but I did get a reply from Sovol about this in a very short time frame, they were fast to respond. I'll keep you posted as any news arrives.

**Reply by ghost** [reply]
Great news, they have already replied (they are so fast!) that they will supply the files soon on their official github: 
https://github.com/Sovol3d/SV06-Fully-Open-Source/issues/2#issuecomment-1441524355

Sovol are really impressive right now.

**Comment by bassamanator** [comment]
That's brilliant! Thanks for keeping all of us posted. Will be great to include this info in this repo!
